templates/archives/stories/time_records.jinja2

```diff
@@ -1,39 +1,44 @@
 {% extends 'archives/story.jinja2' %}
 
 {% block story_title %}
-Using a Supercomputer
+Cost of a Supercomputer
 {% endblock %}
 
 {% block story_body %}
-<p>When requesting time to use the computer, laboratories trying to tackle problems could only give an estimate of how much time they needed. Knowing this, one could assume that many laboratories used more time than they requested while others used less time than they requested. Surprisingly, these estimations were off by thousands of hours.</p>
+<p>When requesting time to use the computer, laboratories could only give an estimate of how
+    much computer time they think they would need. This resulted in absurd costs for solving
```

Comment:
Who is *they*?

Reply:
They is addressing the various laboratories.